### ISSUE TITLE
Module YateaExtractor won't accept relative paths

### ISSUE THREAD

**Issue by mandiayba** [Bibliome/alvisnlp#109]
Module `YateaExtractor` doesn't accept relative paths for `configDir` and `localeDir` params:

logs :
```
[2020-01-27 16:21:07.700][map-microorganisms.term-extraction.yatea] processing
[2020-01-27 16:21:07.701][map-microorganisms.term-extraction.yatea] create-input
[2020-01-27 16:21:07.702][map-microorganisms.term-extraction.yatea] owner.getOutputDir() = null
[2020-01-27 16:21:07.702][map-microorganisms.term-extraction.yatea] options = {monolexical-included=0, TT-for-BioLG=0, debug=1, printChunking=0, output-path=., match-type=, monolexical-all=1, termList=multi, language=EN, TC-for-BioLG=0, xmlout=1, MESSAGE_DISPLAY=EN, XML-corpus-for-BioLG=0, TTG-style-term-candidates=multi, suffix=default, annotate-only=0}
[2020-01-27 16:21:07.702][map-microorganisms.term-extraction.yatea] defaultConfig = {LOCALE_DIR=resources/YaTeA/locale, CONFIG_DIR=resources/YaTeA/config-habitats, RESULT_DIR=$PWD/RESULTS}
[2020-01-27 16:21:07.729][map-microorganisms.term-extraction.yatea] yatea (stderr and stdout)
[2020-01-27 16:21:07.892][map-microorganisms.term-extraction.yatea]
[2020-01-27 16:21:07.892][map-microorganisms.term-extraction.yatea]
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea]
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea]
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] #     #     #    #########  #######     #
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] #   #     # #       #      #          # #
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] # #     #   #      #      #         #   #
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] #     #     #     #      #####    #     #
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] #     #######     #      #        #######
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] #     #     #     #      #        #     #
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] #     #     #     #      #######  #     #
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea]
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea]
[2020-01-27 16:21:07.893][map-microorganisms.term-extraction.yatea] No such repository :resources/YaTeA/config-habitats/EN
[2020-01-27 16:21:07.895][alvisnlp] SEVERE unrecoverable error

### error type:
###     fr.inra.maiage.bibliome.alvisnlp.core.module.ProcessingException
###
### error message:
###     external program returned 2
###
### stack trace:
###     fr.inra.maiage.bibliome.alvisnlp.core.module.lib.ExternalHandler.doExec(ExternalHandler.java:129)
###     fr.inra.maiage.bibliome.alvisnlp.core.module.lib.ExternalHandler.start(ExternalHandler.java:98)
###     fr.inra.maiage.bibliome.alvisnlp.core.module.lib.ExternalHandler.start(ExternalHandler.java:105)
###     fr.inra.maiage.bibliome.alvisnlp.bibliomefactory.modules.yatea.YateaExtractor.process(YateaExtractor.java:45)
###     fr.inra.maiage.bibliome.alvisnlp.bibliomefactory.modules.yatea.YateaExtractor.process(YateaExtractor.java:34)
###     fr.inra.maiage.bibliome.alvisnlp.core.app.cli.CommandLineProcessingContext.processCorpus(CommandLineProcessingContext.java:128)
###     fr.inra.maiage.bibliome.alvisnlp.bibliomefactory.modules.Sequence_Impl.process(Sequence_Impl.java:112)
###     fr.inra.maiage.bibliome.alvisnlp.bibliomefactory.modules.Sequence_Impl.process(Sequence_Impl.java:45)
###     fr.inra.maiage.bibliome.alvisnlp.core.app.cli.CommandLineProcessingContext.processCorpus(CommandLineProcessingContext.java:128)
###     fr.inra.maiage.bibliome.alvisnlp.bibliomefactory.modules.Sequence_Impl.process(Sequence_Impl.java:112)
###     fr.inra.maiage.bibliome.alvisnlp.bibliomefactory.modules.Sequence_Impl.process(Sequence_Impl.java:45)
###     fr.inra.maiage.bibliome.alvisnlp.core.app.cli.CommandLineProcessingContext.processCorpus(CommandLineProcessingContext.java:128)
###     fr.inra.maiage.bibliome.alvisnlp.core.app.cli.AbstractAlvisNLP.process(AbstractAlvisNLP.java:1103)
###     fr.inra.maiage.bibliome.alvisnlp.core.app.cli.AbstractAlvisNLP.run(AbstractAlvisNLP.java:1210)
###     fr.inra.maiage.bibliome.alvisnlp.core.app.cli.AlvisNLP.main(AlvisNLP.java:85)
```

**Comment by rbossy** [comment]
Please pull and test from branch [`issue-109_YateaExtractor-relative-paths`](https://github.com/Bibliome/alvisnlp/tree/issue-109_YateaExtractor-relative-paths), and test.

**Comment by rbossy** [comment]
@mandiayba , is this ok?

**Comment by mandiayba** [comment]
re-conducting the experiment with the branch [issue-109_YateaExtractor-relative-paths](https://github.com/Bibliome/alvisnlp/tree/issue-109_YateaExtractor-relative-paths), let you know soon...

@rbossy

**Comment by mandiayba** [comment]
working fine with the branch, thx

**Comment by rbossy** [comment]
Merged in 8453778a911122c9a87382c9f5be6e7730bec384
